### PR TITLE
Add Oragon Network Testnet

### PR DESCRIPTION
**Pull request by SeftzZz**
Add chain Oragon Network Testnet

### CODE REVIEW

**Review by SeftzZz — COMMENTED** [review]
Add Oragon Network Testnet

**Comment by SeftzZz** [comment]
Hey @ligi, please approve this pull request, thanks.

**Comment by SeftzZz** [comment]
Hey @ligi please approve this request

File: _data/icons/oragonnetwork.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "https://ipfs.io/ipns/k51qzi5uqu5dgpt64bjivhwzlatlcv9i7abe4vr47mwvu42kru5r1ap60q0b9i",
```
Comment (macvts) [inline _data/icons/oragonnetwork.json:3]: this ipfs url format is incorrect one

File: _data/chains/eip155-88808.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "symbol": "ORGN",
+    "decimals": 18
+  },
+  "infoURL": "https://oragon.network",
+  "shortName": "oragonnetwork",
+  "chainId": 88808,
+  "networkId": 88808,
+  "explorers": [
+    {
+      "name": "Oragon Network Explorer",
```
Comment (macvts) [inline _data/chains/eip155-88808.json:18]: If this will be use once you have mainnet launch, prefer using ORGN Explorer Testnet as shown on url below

**Comment by SeftzZz** [comment]
hi @macvts i have update files, please approval, thanks.

**Pull request closed without merging**